apps/web/app/global-error.tsx

```
@@ -0,0 +1,31 @@
+			tags: {
+				component: 'GlobalError',
+				error_boundary: 'root',
+			},
+			extra: {
+				digest: error.digest,
+			},
+		})
+	}, [error])
+
+	return (
+		<html>
```

Biome (2.1.2)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)